src/main/java/emissary/place/VersionPlace.java

```diff
@@ -63,6 +66,10 @@ private void configurePlace() {
 
         includeDate = configG.findBooleanEntry("INCLUDE_DATE", true);
         useAbbrevHash = configG.findBooleanEntry("USE_ABBREV_HASH", true);
+
+        String cfgRegex = configG.findStringEntry("VERSION_REGEX_FOR_NO_DATE", "^(\\d+\\.)?(\\d+\\.)?(\\d+)$");
```

Comment:
Did you want to update VersionPlace.cfg with VERSION_REGEX_FOR_NO_DATE?

Reply:
I see now that INCLUDE_DATE and USE_ABBREV_HASH are not in the cfg file either.  Is there a pattern for when config values are in the cfg and when they are just in the place?

Reply:
I think in general if the default it sufficient we leave it out of the .cfg file. Anyone is free to write their own .cfg file and use a different value which is what we like to allow for.

Reply:
@jpdahlke - sounds, good, thx for the explanation